src/sinks/blackhole.rs

```diff
@@ -17,11 +17,16 @@ pub struct BlackholeSink {
     acker: Acker,
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone)]
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
 pub struct BlackholeConfig {
+    #[serde(default = "default_print_amount")]
```

Comment:
@bruceg recently pointed me to our use of [`Derivative`](https://mcarton.github.io/rust-derivative/) elsewhere which can be used to write this something like:

```
#[derivative(Default)]
#[derive(Clone, Debug, Default, Deserialize, Serialize)]
pub struct BlackholeConfig {
    #[derivative(Default(value="1000"))]
    #[serde(default)]
    pub print_amount: usize,
}
```

To save the need to define a `const fn`.

Reply:
Oh neat. Thanks.

Reply:
So in order to use the value of `print_amount` in internal metrics, I needed to appease serde anyway so it's now a mix of the above.

Reply:
Aha, I thought `#[serde(default)]` would take care of that.